lexoid/core/parse_type/static_parser.py

```diff
@@ -58,6 +72,27 @@ def parse_static_doc(path: str, **kwargs) -> Dict:
             "parent_title": kwargs.get("parent_title", ""),
             "recursive_docs": [],
         }
+    elif "presentation" in file_type:
+        md_path = os.path.join(kwargs["temp_dir"], f"{int(time())}.md")
+        convert(
+            ConversionConfig(
+                pptx_path=path,
+                output_path=md_path,
+                image_dir=None,
+                disable_image=True,
```

Comment:
Any specific reason, why we should set `disable_image` as True?
Maybe it's fine to extract what works for now? Later we can improve as needed?

Reply:
Makes sense. Will create a ticket.